src/locales/en-AU.json

```diff
@@ -223,6 +223,13 @@
   "mnoe_admin_panel.dashboard.product_markups.offers.modal.table_headers.final_prices": "Final Prices",
   "mnoe_admin_panel.dashboard.product_markups.offers.modal.usage_based": "usage based",
   "mnoe_admin_panel.dashboard.product_markups.offers.modal.one_off": "one off",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.title": "User dashboards",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.name": "Name",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.currency": "Currency",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.updated_at": "Updated at",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.created_at": "Created at",
+  "mnoe_admin_panel.dashboard.dashboard_list.widget.list.no_kpis": "This dashboard does not have any kpis.",
```

Comment:
If customers see this, we could format this to "KPIs" instead of "kpis"

Reply:
👍

Reply:
👍